src/ecalc/libraries/libecalc/fixtures/libecalc/fixtures/cases/consumer_system_v2/consumer_system_v2_dto.py

```diff
@@ -173,23 +173,23 @@ def consumer_system_v2_dto() -> DTOCase:
         operational_settings={
             datetime(2022, 1, 1, 0, 0): [
                 dto.components.CompressorSystemOperationalSetting(
-                    rates=[Expression.setup_from_expression(x) for x in [1000000, 5000000, 6000000]],
+                    rates=[Expression.setup_from_expression(x) for x in [1000000, 5000000, 5000000]],
                     inlet_pressures=[Expression.setup_from_expression("50")] * 3,
                     outlet_pressures=[Expression.setup_from_expression("250")] * 3,
-                    crossover=[0, 1, 1],
-                ),
+                    crossover=[0, 0, 0],
+                ),  # Invalid operational setting, should not be valid for any timesteps
                 dto.components.CompressorSystemOperationalSetting(
-                    rates=[Expression.setup_from_expression(x) for x in [500000, 2500000, 3000000]],
+                    rates=[Expression.setup_from_expression(x) for x in [1000000, 5000000, 5000000]],
                     inlet_pressures=[Expression.setup_from_expression("50")] * 3,
                     outlet_pressures=[Expression.setup_from_expression("125")] * 3,
-                    crossover=[0, 1, 1],
+                    crossover=[0, 1, 1],  # Crossover makes this valid. 1 mill from consumer 2 and 3 sent to 1.
```

Comment:
Before this PR this test would have failed because the system did not evaluate with this crossover, instead the crossover in the first operational setting was always used.